source/threads.tex

```diff
@@ -1283,17 +1296,44 @@
 \begin{itemdescr}
 \pnum
 \effects
-Inserts an unspecified text representation of \tcode{id} into
-\tcode{out}. For two objects of type \tcode{thread::id} \tcode{x} and \tcode{y},
-if \tcode{x == y} the \tcode{thread::id} objects have the same text
-representation and if \tcode{x != y} the \tcode{thread::id} objects have
-distinct text representations.
+Inserts the text representation for \tcode{charT} of \tcode{id} into
+\tcode{out}.
 
 \pnum
 \returns
 \tcode{out}.
 \end{itemdescr}
 
+\indexlibrarymember{formatter}{thread::id}%
+\begin{itemdecl}
+template<class charT> class formatter<thread::id, charT>;
```

Comment:
Paper-correct. But it's wrong.
```cli
$ git grep -C 1 'class formatter' *.tex
$ git grep -C 1 'struct formatter' *.tex | wc -l
121
```
```suggestion
template<class charT> struct formatter<thread::id, charT>;
```

Reply:
Notably, it's also a `struct` in the synopsis.

Reply:
Yes, please. (Not core-technically a problem, but we should be consistent.)

Reply:
This is subject of #5323.